AEPIdentity/Sources/Identity+PublicAPI.swift

```diff
@@ -143,3 +156,17 @@ import Foundation
         }
     }
 }
+
+extension CustomIdentity {
```

Comment:
We should move this to CustomIdentity.swift

Reply:
Yes, I wasn't sure whether to place this in CustomIdentity where others could use it, or just where it's needed in Identity+PublicAPI. I'll move it to the CustomIdentity class.